### ISSUE TITLE
latest image not giving latest homebridge?

### ISSUE THREAD

**Issue by andrasg** [oznu/docker-homebridge#487]
### Describe The Bug

Installing the `latest` tag on a pi (`Linux ha-rpi 5.15.74-v8+ #1595 SMP PREEMPT Wed Oct 26 11:07:24 BST 2022 aarch64 GNU/Linux`) gets me version `1.5.0` with `1.6.0` showing on the UI as being available:

<img width="282" alt="image" src="https://user-images.githubusercontent.com/10730249/211022429-b54a59a5-45e2-46a8-9b78-91fdf652436c.png">

Looks like the build process wasn't triggered for `1.6.0`?

### Docker Config

```yml
docker run -d \
  --net=host \
  --name=homebridge \
  --restart unless-stopped \
  -e HOMEBRIDGE_CONFIG_UI=1 \
  -e HOMEBRIDGE_CONFIG_UI_PORT=8080 \
  -e ENABLE_AVAHI=1 \
  -e MQTTTHING_URL="mqtt://192.168.20.15:1883" \
  -v /home/pi/volumemaps/homebridge/config:/homebridge \
  oznu/homebridge
```

### Host Operating System

Raspberry Pi OS - Latest

### Host Architecture

aarch64 / arm64

**Comment by tosate** [comment]
I had the same problem. Look at [this](https://github.com/oznu/docker-homebridge/issues/478#issuecomment-1399569613)

TL;DR

- stop the container
- delete the file package.json
- start the container

**Comment by andrasg** [comment]
Thanks, this actually worked. Makes me think, that probably it is no such good idea to persist the complete `/homebridge` path onto the host when using docker...? Thinking about `node_modules` and other files/folder that might be version dependent.

**Comment by DennisJohnsen** [comment]
> I had the same problem. Look at [this](https://github.com/oznu/docker-homebridge/issues/478#issuecomment-1399569613)
> 
> TL;DR
> 
> * stop the container
> * delete the file package.json
> * start the container

Careful if you do this. This will cause your plugins to be removed (not your config though) so you will have to install the plugins again. Hopefully not having to set it all up in Homekit again - but i havn't gotten to that yet.

**Comment by tosate** [comment]
I have added the plug-ins to the startup.sh script. Isn't this the recommended way to install them?
This way the deletion of the package.json file had no influence on my plug-ins.

```#!/bin/sh

#
# Docker Homebridge Custom Startup Script - oznu/homebridge
#
# This script can be used to customise the environment and will be executed as
# the root user each time the container starts.
#
# If using this to install plugins DO NOT use the global flag (-g).
#
# Example installing homebridge plugin:
#
# npm install homebridge-hue
#
# Example installing packages using default image (Alpine Linux):
#
# apk add --no-cache ffmpeg ffmpeg-libs libpcap-dev
#
# If you're running a debian based version of this container please use apt-get to install packages.
#

#npm install homebridge-knx homebridge-mqttthing homebridge-hue
npm install homebridge-knx@latest homebridge-mqttthing@latest homebridge-hue@latest
```

**Comment by andrasg** [comment]
> I have added the plug-ins to the startup.sh script. Isn't this the recommended way to install them? This way the deletion of the package.json file had no influence on my plug-ins.

This is exactly the way I have it configured. Cleaning `package.json`, and it reinstalled all my plugins on boot using `startup.sh`.
Had no issues.

**Comment by DennisJohnsen** [comment]
> > I have added the plug-ins to the startup.sh script. Isn't this the recommended way to install them? This way the deletion of the package.json file had no influence on my plug-ins.
> 
> This is exactly the way I have it configured. Cleaning `package.json`, and it reinstalled all my plugins on boot using `startup.sh`. Had no issues.



> I have added the plug-ins to the startup.sh script. Isn't this the recommended way to install them? This way the deletion of the package.json file had no influence on my plug-ins.
> 
> ```
> 
> #
> # Docker Homebridge Custom Startup Script - oznu/homebridge
> #
> # This script can be used to customise the environment and will be executed as
> # the root user each time the container starts.
> #
> # If using this to install plugins DO NOT use the global flag (-g).
> #
> # Example installing homebridge plugin:
> #
> # npm install homebridge-hue
> #
> # Example installing packages using default image (Alpine Linux):
> #
> # apk add --no-cache ffmpeg ffmpeg-libs libpcap-dev
> #
> # If you're running a debian based version of this container please use apt-get to install packages.
> #
> 
> #npm install homebridge-knx homebridge-mqttthing homebridge-hue
> npm install homebridge-knx@latest homebridge-mqttthing@latest homebridge-hue@latest
> ```

Thanks to you both. Added my plugins to `startup.sh` :)
